scripts/freshvenvs.py

```diff
@@ -67,10 +86,22 @@ def _get_riot_envs_including_any(modules: typing.Set[str]) -> typing.Set[str]:
 
 
 def _get_packages_implementing(modules: typing.Set[str]) -> typing.Set[str]:
```

Comment:
Because this function now does filtering, consider renaming it to make that clear to the caller (possibly `_get_updateable_packages_implementing`)